e2e/backend.e2e.spec.ts

```diff
@@ -110,10 +110,7 @@ test.describe.serial('KubernetesBackend', () => {
         },
       };
       /** Platform-specific changes to `newSettings`. */
-      const platformSettings: Partial<Record<NodeJS.Platform, RecursivePartial<Settings>>> = {
-        win32:  { virtualMachine: { hostResolver: getAlternateSetting(currentSettings, 'virtualMachine.hostResolver', true, false) } },
-        darwin: { experimental: { virtualMachine: { socketVMNet: getAlternateSetting(currentSettings, 'experimental.virtualMachine.socketVMNet', true, false) } } },
```

Comment:
This was here to test platform-specific settings; can we switch to a different platform-specific one instead, such as `useRosetta`?